### PR TITLE
make default string arguments  in schemas human readable

### PR DESCRIPTION
**Pull request by free25zer**
[jit] String default args get printed as ascii values #25804 
https://github.com/pytorch/pytorch/issues/25804

### CODE REVIEW

**Comment by Krovatkin** [comment]
@eellison could you please take a look when you have a mnt? Seems to be a reasonable solution.

**Review by eellison — APPROVED** [review]
LGTM. Although this change seems benign to me a test would be nice

File: test/test_jit.py [hunk 1]
```diff
@@ -19078,6 +19078,12 @@ def __init__(self, x):
                 def set_non_initialized(self, y):
                     self.bar = y  # can't assign to non-initialized attr
 
+    def test_schema_human_readable(self):
+        with self.assertRaisesRegex(RuntimeError, "nearest"):
```
Comment (eellison) [inline test/test_jit.py]: Why are we matching against "nearest" ?

Comment (free25zer) [reply test/test_jit.py]: In the original issue it was the argument to mode that was printed out as octal:
mode='\156\145\141\162\145\163\164'
When it should just be 
mode='nearest'
So I am matching against nearest to make sure that we are printing it correctly

Comment (eellison) [reply test/test_jit.py]: makes sense, thanks

Comment (free25zer) [reply test/test_jit.py]: I'll add in a comment in the test in a PR

**Pull request closed without merging**